iothub/service/src/ClientApiVersionHelper.cs

```diff
@@ -12,22 +12,12 @@ internal class ClientApiVersionHelper
     {
         private const string ApiVersionQueryPrefix = "api-version=";
         private const string ApiVersionGA = "2016-02-03";
-        private const string ApiVersionDefault = "2019-10-01";
-        private const string ApiVersionLimitedAvailability = "2020-03-13";
-
-        // For import/export devices jobs, a new parameter is available in a
-        // new api-version, which is only available in a few initial regions.
-        // Control access via an environment variable. If a user wishes to try it out,
-        // they can set "EnabledStorageIdentity" to "1". Otherwise, the SDK will still
-        // default to the latest, broadly-supported api-version used in this SDK.
-        internal static bool IsStorageIdentityEnabled => StringComparer.OrdinalIgnoreCase.Equals("1", Environment.GetEnvironmentVariable("EnableStorageIdentity"));
+        private const string ApiVersionDefault = "2020-03-13";
```

Comment:
Not related to this change, but why do we have a 2016 service API version supported here? Can we test if all operations can support the latest service API version?

Reply:
Good question. I'll remove it and see if the tests still pass as expected. I don't see why we should still have this reference